QuickFlicks/Controller/MovieDetailViewController.swift

```diff
@@ -9,71 +9,141 @@
 import UIKit
 import Alamofire
 import AlamofireImage
+import SteviaLayout
 
-class MovieDetailViewController: UIViewController, UIScrollViewDelegate {
-
-    @IBOutlet weak var scrollView: UIScrollView!
+class MovieDetailView: UIView {
     
-    @IBOutlet weak var posterImageView: UIImageView!
+    var posterImageView = UIImageView()
+    var contentView = UIView()
+    var titleLabel = UILabel()
+    var dateLabel = UILabel()
+    var rateLabel = UILabel()
+    var timeLabel = UILabel()
+    var overviewLabel = UILabel()
     
-    @IBOutlet weak var titleLabel: UILabel!
+    convenience init() {
+        self.init(frame: CGRectZero)
+        
+        render()
+    }
     
-    @IBOutlet weak var dateLabel: UILabel!
+    func render() {
+        
+        backgroundColor = .whiteColor()
+        
+        // Setup UI, need to clean up later
+        // === BEGIN ===
+        posterImageView.contentMode = .ScaleAspectFill
+        
+        contentView.backgroundColor = UIColor.blackColor()
+        contentView.alpha = 0.8
+        
+        titleLabel.textColor = UIColor.whiteColor()
+        titleLabel.numberOfLines = 0
+        titleLabel.font = UIFont.boldSystemFontOfSize(17)
+        dateLabel.textColor = UIColor.whiteColor()
+        dateLabel.font = dateLabel.font.fontWithSize(13)
+        rateLabel.textColor = UIColor.whiteColor()
+        rateLabel.font = rateLabel.font.fontWithSize(13)
+        timeLabel.textColor = UIColor.whiteColor()
+        timeLabel.font = timeLabel.font.fontWithSize(13)
+        overviewLabel.textColor = UIColor.whiteColor()
+        overviewLabel.font = overviewLabel.font.fontWithSize(13)
+        overviewLabel.numberOfLines = 0
+        // === END ===
+        
+        sv(
+            posterImageView.sv(
+                contentView.sv(
+                    titleLabel,
+                    dateLabel,
+                    rateLabel,
+                    timeLabel,
+                    overviewLabel
+                )
+            )
+        )
+        
+        layout(
+            0,
+            |posterImageView|,
+            0
+        )
+        
+        posterImageView.layout(
+            |-32-contentView.height(150)-32-|,
+            0
+        )
+        
+        contentView.layout(
+            0,
+            |-8-titleLabel-8-|,
+            5,
+            |-8-dateLabel-8-|,
+            20,
+            |-8-rateLabel-timeLabel-8-|,
+            5,
+            |-8-overviewLabel-8-|,
+            0
+        )
+    }
     
-    @IBOutlet weak var rateLabel: UILabel!
+    func fieldStyle(f:UITextField) {
+        f.borderStyle = .RoundedRect
+        f.font = UIFont(name: "HelveticaNeue-Light", size: 26)
+        f.returnKeyType = .Next
+    }
     
-    @IBOutlet weak var timeLabel: UILabel!
+    func passwordFieldStyle(f:UITextField) {
```

Comment:
lol i think you took this from the docs, I just mentioned them above. you don't use these functions anywhere though.